Model gives an error with empty package list

Input:
```
[
    {
        "ecosystem": "maven",
        "package_list": []

    }
]
```
Output:
```
hpf-insights_1  | 2018-08-17 15:11:20,178 flask.app [ERROR] Exception on /api/v1/companion_recommendation [POST]
hpf-insights_1  | Traceback (most recent call last):
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask/app.py", line 2292, in wsgi_app
hpf-insights_1  |     response = self.full_dispatch_request()
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask/app.py", line 1815, in full_dispatch_request
hpf-insights_1  |     rv = self.handle_user_exception(e)
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask_cors/extension.py", line 161, in wrapped_function
hpf-insights_1  |     return cors_after_request(app.make_response(f(*args, **kwargs)))
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask/app.py", line 1718, in handle_user_exception
hpf-insights_1  |     reraise(exc_type, exc_value, tb)
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask/_compat.py", line 35, in reraise
hpf-insights_1  |     raise value
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask/app.py", line 1813, in full_dispatch_request
hpf-insights_1  |     rv = self.dispatch_request()
hpf-insights_1  |   File "/usr/lib64/python3.4/site-packages/flask/app.py", line 1799, in dispatch_request
hpf-insights_1  |     return self.view_functions[rule.endpoint](**req.view_args)
hpf-insights_1  |   File "/src/flask_endpoint.py", line 92, in hpf_scoring
hpf-insights_1  |     input_stack['package_list'])
hpf-insights_1  |   File "/src/scoring/hpf_scoring.py", line 197, in predict
hpf-insights_1  |     if len(missing_packages) / len(input_stack) < UNKNOWN_PACKAGES_THRESHOLD:
hpf-insights_1  | ZeroDivisionError: division by zero


```

--- a/src/scoring/hpf_scoring.py
+++ b/src/scoring/hpf_scoring.py
@@ -185,6 +185,8 @@ class HPFScoring:
         missing_packages = set()
         package_topic_dict = {}
         companion_recommendation = []
+        if not input_stack:
+            return companion_recommendation, package_topic_dict, list(missing_packages)
         for package_name in input_stack:
             package_id = self.package_id_dict.get(package_name)
             if package_id:

--- a/tests/scripts/entrypoint-test.sh
+++ b/tests/scripts/entrypoint-test.sh
@@ -18,4 +18,4 @@ echo "*****************************************"
 echo "*** Unit tests ***"
 echo "*****************************************"
 
-pytest --cov=/src/ --cov-report term-missing -vv /tests/unit_tests/
+PYTHONHASHSEED=2 pytest --cov=/src/ --cov-report term-missing -vv /tests/unit_tests/

--- a/tests/unit_tests/test_scoring_hpf_scoring.py
+++ b/tests/unit_tests/test_scoring_hpf_scoring.py
@@ -158,6 +158,12 @@ class TestHPFScoringMethods(unittest.TestCase):
         assert package_set == expected_package_set
         self.assertListEqual(predict_result[2], [])
         assert len(predict_result[1]) == 4
+        input_stack = []
+        predict_result1 = self.hpf_obj.predict(input_stack)
+        predict_result2 = self.hpf_obj_feedback.predict(input_stack)
+        assert not predict_result1[0] and not predict_result2[0]
+        assert not predict_result1[1] and not predict_result2[1]
+        assert not predict_result1[2] and not predict_result2[2]
 
     def test_model_details(self):
         """Test the basic model details function."""

